ui/src/components/SignoffSummary/index.jsx

```diff
@@ -46,24 +61,83 @@ function SignoffSummary(props) {
+          const AllNotSigned = () => {
+            const leftarr = [];
+            const allLeft = count - AllSigned.length;
+
+            // Disabled eslint because "i+1" runs loop till eternity
+            // and for in loop only works on arrays
+            // eslint-disable-next-line no-plusplus
+            for (let i = 0; i < allLeft; i++) {
+              leftarr.push([]);
+            }
+
+            return leftarr;
+          };
 
           return (
-            <ListItem key={key} className={classes.signoffsList}>
-              <ListItemText
-                primary={
-                  <Typography component="p" variant="body2">
-                    {`${count} member${count > 1 ? 's' : ''} of ${role}`}
-                  </Typography>
-                }
-                className={classes.listItemText}
-              />
-            </ListItem>
+            <div key={key} className={classes.signoffsContainer}>
+              <ListItem className={classes.signoffsList}>
+                <ListItemText
+                  primary={
+                    <Typography component="p" variant="body2">
+                      {`${count} member${count > 1 ? 's' : ''} of ${role}  -`}
+                    </Typography>
+                  }
+                  className={classes.listItemText}
+                />
+                {listOfSignoffs && (
+                  <React.Fragment>
+                    {AllSigned.map((arr, index) => {
+                      const no = index;
+
+                      return (
+                        <div key={no} className={classes.approval}>
+                          <CheckCircleIcon color="green" />
+                          <Typography
+                            component="p"
+                            variant="body2"
+                            style={{ color: 'green', width: 'max-content' }}>
+                            {`${arr[0]}`}
+                          </Typography>
+                        </div>
+                      );
+                    })}
+                    {AllNotSigned().map(arr => {
+                      const no = arr;
+
+                      return (
+                        <div key={no} className={classes.approval}>
+                          <AccountClockIcon color="darkorange" />
+                          <Typography
+                            component="p"
+                            variant="body2"
+                            style={{ color: 'darkorange' }}>
+                            Awaiting approval...
+                          </Typography>
+                        </div>
+                      );
+                    })}
+                  </React.Fragment>
+                )}
+              </ListItem>
+            </div>
           );
         })}
       </List>
-      {listOfSignoffs && Boolean(listOfSignoffs.length) && (
+      {/* {listOfSignoffs && Boolean(listOfSignoffs.length) && (
```

Comment:
Please remove this block instead of commenting it out.

Reply:
I'll do that right away. I left the block there while waiting for your comments. I'll take it out ASAP

Reply:
All changes requested are implemented now